ci-operator/config/identitatem/idp-mgmt-operator/identitatem-idp-mgmt-operator-main.yaml

```diff
@@ -0,0 +1,49 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  open-cluster-management_builder_go1.16-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+binary_build_commands: go build ./cmd/manager
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    open-cluster-management_builder_go1.16-linux:
+      as:
+      - registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux
+  to: idp-mgmt-operator
+promotion:
+  name: "0.1"
+  namespace: identitatem
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 80m
+      memory: 200Mi
+tag_specification:
+  name: "4.8"
+  namespace: ocp
+tests:
```

Comment:
Edit: Ignore this for now. I need to set up identitatem in Sonar and I need to make sure the CICD bot account has been created in GitHub.

For your main branch, you also need pre- and post-submit sonar scans and you need the fast forwarding job.

https://github.com/open-cluster-management/cicd-docs/blob/main/prow/onboarding.md#add-sonarcloud-scans

https://github.com/open-cluster-management/cicd-docs/blob/main/prow/onboarding.md#add-the-fast-forward-post-submit-job